### PR TITLE
Update main links in prep of gdpr

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: front/style/layout/_footer.scss [hunk 1]
```diff
@@ -3,6 +3,12 @@ footer {
         width: 190px;
     }
 
+    .donate-mail {
+        font-family: $font-roboto;
+        margin-bottom: 20px;
+    }
+
```
Comment (ottaviano) [inline front/style/layout/_footer.scss]: trailing line

File: templates/membership/join.html.twig [hunk 2]
```diff
@@ -135,7 +135,8 @@
...
                                 <label class="form form__label" for="{{ form.conditions.vars.id }}">
                                     <span class="text__checkbox">
-                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site
+                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site et à la <a href="https://en-marche.fr/politique-protection-donnees" target="_blank">politique de protection des données à caractère personnel</a> du site
+
```
Comment (Remg) [inline templates/membership/join.html.twig]: extra line

Comment (marineb) [reply templates/membership/join.html.twig]: Fixed !

File: templates/membership/join.html.twig [hunk 3]
```diff
@@ -135,7 +135,7 @@
...
                                 <label class="form form__label" for="{{ form.conditions.vars.id }}">
                                     <span class="text__checkbox">
-                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site
+                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site et à la <a href="https://en-marche.fr/politique-protection-donnees" target="_blank">politique de protection des données à caractère personnel</a> du site
```
Comment (ottaviano) [inline templates/membership/join.html.twig]: is it static page `https://en-marche.fr/politique-protection-donnees`? can you use `path('route_name')` to generate the URL?

**Pull request merged**